sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/CBDE/CbdeHandler.cs

```diff
@@ -366,27 +371,37 @@ private void RaiseIssuesFromJSon(CompilationAnalysisContext context)
                 jsonFileContent = File.ReadAllText(cbdeJsonOutputPath);
                 jsonIssues = JsonConvert.DeserializeObject<List<List<JObject>>>(jsonFileContent);
             }
-            catch
+            catch(Exception exception)
             {
-                LogIfFailure($"- error parsing json file {cbdeJsonOutputPath}");
-                Log(logStringBuilder.ToString());
-                Console.Error.WriteLine($"Error parsing output from CBDE, more details in {cbdeProcessSpecificPath}");
-                return;
+                if (exception is JsonException || exception is IOException)
+                {
+                    LogIfFailure($"- error parsing json file {cbdeJsonOutputPath}: {exception.ToString()}");
+                    Log(logStringBuilder.ToString());
+                    Console.Error.WriteLine($"Error parsing output from CBDE, more details in {cbdeProcessSpecificPath}");
```

Comment:
don't forget to replace this with throwing an exception, if you want something to show in the users console (AFAIK, Console output will be eaten by roslyn)

Reply:
Maybe it shows in unit tests, which would be better than nothing?
Otherwise, do you know a way to log info to user?

Reply:
removed and replaced by throwing an exception